examples/phabricator/README.md

```diff
@@ -110,34 +132,48 @@ To automate this process and make sure that a proper host is authorized even if
 
 ```js
 {
-  "id": "authenticator-controller",
   "kind": "ReplicationController",
-  "apiVersion": "v1beta1",
-  "desiredState": {
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "authenticator-controller",
+    "labels": {
+      "name": "authenticator"
+    }
+  },
+  "spec": {
     "replicas": 1,
-    "replicaSelector": {"name": "authenticator"},
-    "podTemplate": {
-      "desiredState": {
-        "manifest": {
-          "version": "v1beta1",
-          "id": "authenticator-pod",
-          "containers": [{
+    "selector": {
+      "name": "authenticator"
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "name": "authenticator"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
             "name": "authenticator",
-            "image": "kubernetes/example-cloudsql-authenticator",
```

Comment:
@nikhiljindal - It appears that the readme listed the wrong image. This json matches the actual image value in both the v1beta1 and v1beta3 json files - https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/phabricator/authenticator-controller.json